### PR TITLE
Add number days after initial census collection to schema 

### PR DESCRIPTION
**Pull request by alanren2**
The rates dict struct now goes: year -> date -> list of rate and days after.

### CODE REVIEW

File: backend/api/views/populate_db.py [hunk 1]
```diff
@@ -19,11 +20,12 @@ def is_county(t):
     return t == COUNTY
 
 
-def parse_census_data(link, date, parse2000=False):
+def parse_census_data(link, date, date_initial, parse2000=False):
     result = requests.get(link).text
     rows = result.strip().split("\n")
     responses = []
     id_to_county = {}
+    date_initial_obj = datetime.date(int(date_initial[-4:]), int(date_initial[:2]), int(date_initial[2:4]))
```
Comment (chyku) [inline backend/api/views/populate_db.py]: could you use `datetime.strptime('').date()` instead- it'd be more readable

File: backend/api/views/populate_db.py [hunk 2]
```diff
@@ -40,18 +42,22 @@ def parse_census_data(link, date, parse2000=False):
         county_id = i[:5]
 
         if validate_data(i, name, t, rates[-1]):
+            date_obj = datetime.date(int(date[-4:]), int(date[:2]), int(date[2:4]))
```
Comment (chyku) [inline backend/api/views/populate_db.py]: same as above for parsing str into date obj

File: backend/api/views/populate_db.py [hunk 3]
```diff
@@ -40,18 +42,22 @@ def parse_census_data(link, date, parse2000=False):
...
+            delta = date_obj - date_initial_obj
             censusResp = CensusResponse(
                 tract_id=i,
                 county=id_to_county[county_id],
-                rates={"2010": {date: rates[-1]}},
+                rates={"2010": {date: [rates[-1], delta.days]}},
```
Comment (chyku) [inline backend/api/views/populate_db.py:50]: bruh rip our db structure but its ok for mvp i guess

**Review by chyku — CHANGES_REQUESTED** [review]
1 thing for readability :ok_hand:

**Review by chyku — APPROVED** [review]
:ok_hand:

**Review by caiyingshen — APPROVED** [review]
😱

**Pull request merged**